staging/src/k8s.io/kubectl/pkg/cmd/get/get.go

```diff
@@ -513,14 +518,9 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 		objs[ix] = infos[ix].Object
 	}
 
-	sorting, err := cmd.Flags().GetString("sort-by")
-	if err != nil {
-		return err
-	}
-
 	var positioner OriginalPositioner
 	if o.Sort {
-		sorter := NewRuntimeSorter(objs, sorting)
+		sorter := NewRuntimeSorter(objs, *o.PrintFlags.HumanReadableFlags.SortBy)
```

Comment:
Here `PrintFlags`, `HumanReadableFlags` and `SortBy` can all be nil, and I don't see you checking any of these, although theoretically when `o.Sort` is true it shouldn't happen, but I wouldn't want to rely on it as such. 
I'd also suggest adding test cases around this place, since it looks like we don't have any, ensuring that any value will work correctly.